src/paths/openapi/transform.yaml

```diff
@@ -0,0 +1,65 @@
+summary: Transform an OpenAPI File into connections and components
+tags:
+  - openapi
+operationId: openapiTransform
+security:
+  - bearerAuth: ["OPENAPI_READ"]
+requestBody:
+  description: The openapi file to be transformed
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+    multipart/formData:
+      schema:
+        type: string
+        format: binary
+responses:
+  '200':
+    description: Expected response to a valid request
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            components:
+              type: array
+              items:
+                $ref: "../../schemas/components/ComponentDto.yaml"
+            connections:
+              type: array
+              items:
+                $ref: "../../schemas/connections/ConnectionDto.yaml"
+  "401":
+    $ref: "../../responses/TokenResponse.yaml"
+  "403":
+    $ref: "../../responses/openapi/ResponseOpenApiForbidden.yaml"
+  '404':
+    description: Expected response to an unexisting openapi file from a server
+    content:
+      application/json:
+        schema:
+          $ref: "../../schemas/response/ErrorResponse.yaml"
+        example:
+          timestamp: "2021-02-09T07:57:01.062+0000"
+          status: 404
+          error: "Not Found"
+          message: "Unable the find the openapi specification"
+          path: "/api/openapi/"
+  '500':
+    description: Expected response to an internal error.
+    content:
+      application/json:
+        schema:
+          $ref: "../../schemas/response/ErrorResponse.yaml"
+        example:
+          timestamp: "2021-02-09T07:57:01.062+0000"
+          status: 500
+          error: "ComponentParser Exception"
+          message: "The openapi title is missing"
+          path: "/api/openapi/"
```

Comment:
should not occur, this should be captured as a 400 with a decent error message.
Is this already handled in the code? Otherwise have a look at ControllerAdvice on how to handle Exceptions in Spring